src/main/java/com/ryandw11/structure/ignoreblocks/IgnoreBlocks.java

```diff
@@ -2,8 +2,8 @@
 
 import org.bukkit.Material;
 
-import java.util.List;
+import java.util.EnumSet;
 
 public interface IgnoreBlocks {
-    List<Material> getBlocks();
+    EnumSet<Material> getBlocks();
```

Comment:
In the future, this should be a separate PR. Try to keep PRs related to specific changes.